### ISSUE TITLE
Maintenance

### ISSUE THREAD

**Issue by edygar** [camwest/react-slot-fill#37]
Hi there!

This repository is currently outdated. My suggestion is to split the demo package from the lib itself so they can be updated at a different pace.

Also, now there's the opportunity to play with the new Hooks API, I would like this library to go forward. 

As part of the [Contributing guide](https://github.com/camwest/react-slot-fill/blob/master/CONTRIBUTING.md), I'm an Intent to implement.

**Comment by veeramarni** [comment]
Yup, I would also like to test Hooks API with this component. @edygar have you done any test with it?

**Comment by dmr** [comment]
@edygar @veeramarni Did you find a solution/maintained project yet?

I'm still using this but I miss the hook :)

**Comment by dev-bjoern** [comment]
Anybody knows some maintained alternatives to this project? I found https://github.com/robik/react-view-slot which seems similar but unknown.

**Comment by exah** [comment]
Hey @chickenwing, I recently published [`nano-slots`](https://github.com/exah/nano-slots) - a modern lightweight alternative with similar component based API. Will be glad to hear your opinion :)
